.gitignore

```diff
@@ -1,6 +0,0 @@
-vendor
```

Comment:
Why delete this file?

Reply:
It's automatically generated when the .goreleaser file is generated with items not required since we have custom goreleaser files and are only using the default for release-notes generation.

Reply:
We shouldn't delete our gitignore file